content/surveys/_index.en.md

```diff
@@ -7,12 +7,13 @@ weight: 15
 
 # Survey Templates
 
-The survey templates have been developed to match the data that would be collected using MSF Ethical Review Board pre approved surveys ([available here](https://fieldresearch.msf.org/handle/10144/618942)) on: 
-* ***retrospective mortality and access to care***, 
-* ***vaccination coverage*** and 
-* ***malnutrition***. 
+The survey templates have been developed to match the data that would be collected using MSF Ethical Review Board pre approved surveys ([MSF Ethical Review Board pre-approved surveys](https://fieldresearch.msf.org/handle/10144/618942)) on: 
```

Comment:
There is no need for the parenthesis or the repetition here. It's expected to include the link in text (wikipedia style)